src/Admin/AbstractAdmin.php

```diff
@@ -2235,32 +2124,32 @@ public function isGranted($name, $object = null)
         return $this->cacheIsGranted[$key];
     }
 
-    public function getUrlSafeIdentifier($entity)
+    public function getUrlSafeIdentifier($entity): ?string
```

Comment:
```suggestion
    public function getUrlSafeIdentifier(object $entity): ?string
```

Reply:
I think you've made this suggestion before.
Please, note that this argument is passed to the `getUrlSafeIdentifier()` method at the model manager, which seems to allow `null`.